src/peterportal.types.ts

```diff
@@ -1,129 +0,0 @@
-// Ripped straight from here: https://github.com/icssc/peterportal-public-api/blob/c539a62acfbfda1481152272582632a168f34302/types/websoc.types.ts
```

Comment:
Why is this file deleted?

Reply:
I must have accidentally selected it when trying to delete `react-app-env.d.ts`